.eslintrc.json

```diff
@@ -16,7 +16,6 @@
   ],
   "plugins": ["@typescript-eslint", "prettier", "react", "react-hooks"],
   "rules": {
-    "prettier/prettier": ["error"],
```

Comment:
This should not be removed thought 😅

Reply:
oops, I undid that change now so it should match the eslintrc.json that's currently in the main branch! Thanks for flagging.